### DISCUSSION TITLE
send-reply hides the comment content and atk-content p some display errors?

### DISCUSSION THREAD

**Discussion by bibicadotnet** [ArtalkJS/Artalk#644] [General]
![2023-11-17_3-55-28](https://github.com/ArtalkJS/Artalk/assets/119997727/08b9468d-1d0b-455e-b6ab-fe805d19356d)
1. The situation is quite strange, because I don't think anyone has reported this problem?

For some reason, when you type a comment into the form, when you reach the bottom right, these words will be covered by the reply button.

I temporarily solved it by moving the position of this reply button to the bottom left, it's quite good on large screen devices, but on small devices like phones, the buttons will not be large enough, so I changed it. limit reuse of default css to size < 768px
Not too optimized on mobile devices, hopefully the authors can handle this issue

```
.atk-send-reply {
float: left;
 position: sticky !important;
margin-top: 5px !important;
	margin-left: 10px !important;
}
@media only screen and (max-width: 768px) {
  .atk-send-reply {
position: absolute !important;
}
}
```
![2023-11-17_4-07-51](https://github.com/ArtalkJS/Artalk/assets/119997727/fa43a09e-8e85-42a8-b8c8-c8a276ec17ff)

https://github.com/ArtalkJS/Artalk/assets/119997727/576ec506-d550-4f7d-855f-e38a72ea8364

2. The way to break words at the end of a sentence is quite strange

~~Inside the comment form the word breaks are displayed correctly~~ 
(ah, I was a bit mistaken, atk-textarea also needs adjustment), but in` atk-content p` there are no word breaks
![2023-11-17_4-15-41](https://github.com/ArtalkJS/Artalk/assets/119997727/b58687f2-1fcb-44ab-815b-9b288d717058)

Small adjustment with 1 piece of css
```
.atk-main-editor>.atk-textarea-wrap>.atk-textarea {
	word-wrap: break-word !important;
}
.atk-content p {
word-break: break-word !important;
}
```

![2023-11-17_4-18-22](https://github.com/ArtalkJS/Artalk/assets/119997727/8d298b6c-fec8-4eca-bdbe-dfe8c195780c)

At this point, the comment content has correct word spacing

Hopefully the author will update and fix some of these small errors in the next versions

**Comment by qwqcode** [comment]
1. The later version will optimize CSS to address this overlapping issue. 
2. Indeed, adding the `word-wrap: break-word` CSS property to paragraphs would enhance the layout and improve readability.

In subsequent versions, these updates will be included. Thank you for your feedback!

**Reply by bibicadotnet** [reply]
I think you forgot this topic

**Reply by qwqcode** [reply]
Hello, version 2.8.2 will include these changes. I've optimized the styles. 😀

<img width="661" alt="image" src="https://github.com/ArtalkJS/Artalk/assets/22412567/f2b70e84-d9b7-4a1e-af43-42f3899bbc02">

**Reply by bibicadotnet** [reply]
I'm actually very tired of the interface on mobile devices

![z5138827266492_8327be85dc13b4b70bf5fd8425b93290](https://github.com/ArtalkJS/Artalk/assets/119997727/56cfdbeb-d2e4-4af9-952d-367377157ed0)
Because avatars are included, the content is always skewed to the right quite a bit

Basically, it's quite good in comment deep 1, but from comment deep 2 onwards, the content almost loses 1/2 of the display area.

![z5138829352597_deba9cf7e0be119b4d498d5159a1ba39](https://github.com/ArtalkJS/Artalk/assets/119997727/7677ea66-c367-4a07-8a07-b37fb9bd54a7)
And if someone replies to comment deep 2, it doesn't even display enough text

![z5138850365110_1492fb91f5f15515e8d07fd1e4d2df10](https://github.com/ArtalkJS/Artalk/assets/119997727/83ba7d33-43cc-4873-9dca-d260b7f56f65)
Can the interface on mobile be made similar to github, easy enough to observe, and full of displayed content?
